apps/test/unit/templates/curriculumCatalog/curriculumCatalogTest.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
+
+describe('CurriculumCatalogCard', () => {
+  let defaultProps;
+  beforeEach(() => {
+    defaultProps = {
+      courseDisplayName: 'AI for Oceans',
+      duration: 'quarter',
+      youngestGrade: 4,
+      oldestGrade: 12,
+      subjects: ['math'],
+      topics: ['cybersecurity']
+    };
+  });
+
+  it('renders course name', () => {
+    render(<CurriculumCatalogCard {...defaultProps} />);
+
+    screen.getByRole('heading', {name: defaultProps.courseName});
```

Comment:
All the `getBy...` queries throw an error if there are 0 matches or more than 1 match. See https://testing-library.com/docs/queries/about/#types-of-queries. Thus, these queries are similar to Enzyme's `expect(wrapper.find('.lastSavedMessage').length).to.equal(1);`

Reply:
You can see a drone run with a failing test here https://drone.cdn-code.org/code-dot-org/code-dot-org/27425/1/4